stylus/settings/palette.styl

```diff
@@ -0,0 +1,84 @@
+// Colors
+//
+// Colors used in the user interface. You can directly use the var name assiociated with its hexadecimal.
+//
+// Styleguide Settings.colors
+
+// Grey
+//
+// white          -  #FFFFFF
+// pale-grey      -  #F5F6F7
+// silver         -  #D6D8Da
+// cool-grey      -  #95999D
+// slate-grey     -  #5D6165
+// charcoal-grey  -  #32363F
+// black          -  #000000
+//
+// Styleguide Settings.colors.grey
+pale-grey     = #F5F6F7
+silver        = #D6D8Da
+cool-grey     = #95999D
+slate-grey    = #5D6165
+charcoal-grey = #32363F
+
+
+// Blue
+//
+// dodger-blue   -  #297EF2
+// science-blue  -  #0B61D6
+// puerto-rico   -  #4DCEC5
+//
+// Styleguide Settings.colors.blue
+dodger-blue  = #297EF2
+science-blue = #0B61D6
+puerto-rico  = #4DCEC5
+
+// Green
+//
+// emerald    -  #35CE68
+// malachite  -  #08b442
+//
+// Styleguide Settings.colors.green
+emerald   = #35CE68
+malachite = #08b442
+
+// Orange
+//
+// texas-rose  -  #FFAE5F
+//
+// Styleguide Settings.colors.orange
+texas-rose = #FFAE5F
+
+// Red
+//
+// your-pink    -  #FDCBCB
+// pomegranate  -  #F52D2D
+// monza        -  #DD0505
+//
+// Styleguide Settings.colors.red
+your-pink   = #FDCBCB
+pomegranate = #F52D2D
+monza       = #DD0505
+
+// Purple
+//
+// portage  -  #9169F2
+//
+// Styleguide Settings.colors.purple
+portage = #9169F2
+
+// 50 (well, in fact, mostly 12) shades of grey
+// /!\ Depreciated colors with fallback for a time
+grey-01 = pale-grey
+grey-02 = silver
+grey-03 = silver
+grey-04 = cool-grey
+grey-05 = cool-grey
+grey-06 = cool-grey
+grey-07 = slate-grey
+grey-08 = charcoal-grey
+grey-09 = silver
+grey-10 = pale-grey
+grey-11 = cool-grey
+grey-12 = slate-grey
```

Comment:
it's been a long time since those greys are deprecated. let's remove them for good.